recipe_scrapers/marleyspoon.py

```diff
@@ -65,6 +66,18 @@ def _get_json_params(self):
         if api_url is None or api_token is None:
             raise ElementNotFoundInHtml("Required script not found.")
 
+        scraper_name = self.__class__.__name__
+        expected_domain = scraper_name.lower()
+        try:
+            api_url = urljoin(self.url, api_url)
```

Comment:
Hrm; sketchy use of variable name shadowing here.  I'll modify this to use a different variable name so that existing behaviour of the scraper is not modified.